infra/base-images/base-builder/Dockerfile

```diff
@@ -83,7 +83,7 @@ ENV SANITIZER_FLAGS_dataflow "-fsanitize=dataflow"
 
 ENV SANITIZER_FLAGS_thread "-fsanitize=thread"
 
-ENV SANITIZER_FLAGS_introspector "-flegacy-pass-manager -flto"
+ENV SANITIZER_FLAGS_introspector "-flegacy-pass-manager -flto -fno-gnu-inline-asm -fno-inline-functions"
```

Comment:
No idea why github thinks I'm making this change.

Reply:
https://github.com/google/oss-fuzz/commit/ba7f8eb9bfd49eba410cef8ae73b2cc4a1ac8cc5